src/main/java/com/bobi89/medicalclinic/controller/DoctorController.java

```diff
@@ -38,4 +39,14 @@ public DoctorDTO save(@RequestBody DoctorDTOwithPassword doctorDTOwithPassword)
     public DoctorDTO addLocationToDoctor(@RequestBody long locationId, @PathVariable long doctorId) {
         return doctorService.addLocationToDoctor(locationId, doctorId);
     }
+
+    @PostMapping("/{doctorId}/scheduleSlot")
```

Comment:
Jako, że jest to dodanie nowej wizyty zrobiłbym nowy controller VisitController i tam udostepnil funkcjonalnosc zwiazane z zarządzaniem wizytami